app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -373,19 +373,34 @@ class ThreadListAdapter @Inject constructor(
     }
 
     private fun CardviewThreadItemBinding.formatRecipientNames(recipients: List<Recipient>): String {
+
+        fun everyone(): String {
+
+            var everyone = ""
+            var isFirstMe = true
+
+            recipients.forEach { recipient ->
+
+                val name = recipient.displayedName(context)
+
+                val formattedName = when {
+                    recipient.isMe() -> (if (isFirstMe) name else null)?.also { isFirstMe = false }
```

Comment:
Wouldn't it be clearer to just make filtered recipients by their email addresses to avoid having several time the same ?

Reply:
Well… That's a thing I want since Day 1 of handling Contacts in this app, but… Good luck trying to convince people.